cryptofeed/exchange/bitmex.py

```diff
@@ -55,7 +55,8 @@ def _parse_symbol_data(cls, data: dict, symbol_separator: str) -> Tuple[Dict, Di
         return ret, info
 
     def __init__(self, **kwargs):
-        super().__init__('wss://www.bitmex.com/realtime', **kwargs)
+        auth_api = 'wss://www.bitmex.com/realtime' if not sandbox else 'wss://testnet.bitmex.com/realtime'
+        super().__init__(f'{auth_api}/realtime', **kwargs)
```

Comment:
looks like
        super().__init__(f'{auth_api}/realtime', **kwargs)
should be
        super().__init__(auth_api, **kwargs)

Reply:
Modification added. Thank you for your help!